src/components/ValidatedInput/ValidSelect.tsx

```diff
@@ -79,6 +83,9 @@ export const ValidSelect = ({
       dataQaProps={selectDataQaProps}
     />
     {isInvalid && <FieldErrorMessage message={errMessage} />}
+    {isShowReloadWhileFetchFailed && (
```

Comment:
there is only one place that this is used (SubscriptionContentsAddModal). how about we add this FieldReloadMessage in that component?
what i mean it'll be something like
<ValidSelect ... />
<FieldReloadMessage ... />

Reply:
I have no problem with that, but I think it has potential to break the styling, wdyt?

Reply:
i'm fine with that. what's different will be FieldReloadMessage will be below helper text which doesn't seem to be a big problem. plus, SubscriptionContentsAddModal doesn't have helpertext